great_expectations/rule_based_profiler/data_assistant/data_assistant_runner.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional, Type, Union
+
+from great_expectations.core import ExpectationSuite
+from great_expectations.core.batch import BatchRequestBase
+from great_expectations.rule_based_profiler.data_assistant import DataAssistant
+from great_expectations.rule_based_profiler.helpers.util import (
+    get_validator_with_expectation_suite,
+)
+from great_expectations.rule_based_profiler.types.data_assistant_result import (
+    DataAssistantResult,
+)
+from great_expectations.validator.validator import Validator
+
+
+class DataAssistantRunner:
+    """
+    DataAssistantRunner processes invocations of calls to "run()" methods of registered "DataAssistant" classes.
+
+    The approach is to instantiate "DataAssistant" class of specified type with "Validator", containing "Batch" objects,
+    specified by "batch_request", loaded into memory.  Then, "DataAssistant.run()" is issued with given directives.
+    """
+
```

Comment:
Do we have this in the design or spec anywhere? Do we really need an entirely new class for all this? Would love to get the team's thoughts on ergonomics.

Reply:
@cdkini This is an internal implementation that makes the required pattern work (per the JIRA item).  This is very akin to `validator.expect_*`.  Thanks.